addons/mail/data/mail_data.xml

```diff
@@ -68,5 +68,32 @@
             <field name="name">Discussions</field>
             <field name="sequence" eval="0"/>
         </record>
+
+        <record id="action_server_module_install_mail_send" model="ir.actions.server">
+            <field name="name">Module Immediate Install Mail Send</field>
+            <field name="condition">True</field>
+            <field name="type">ir.actions.server</field>
+            <field name="model_id" ref="base.model_ir_module_module" />
+            <field name="state">code</field>
+            <field name="code">module_id = context.get('active_id')
```

Comment:
hello @antonylesuisse , Do you mean by move this code to module.py in separate method in base module and call the method from  here ??  if yes? what about mail_template 's send_mail here in below code?  so i think it's batter to be in mail because without mail installed this code(method) is useless.  OR move this code to separate file module.py in mail module ??  Thanks